group_hooks.go

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright 2020, Eric Stevens
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"fmt"
+	"time"
+)
+
+// GroupHook represents a GitLab Group Hook
+type GroupHook struct {
+	ID                       int        `url:"id,omitempty" json:"id,omitempty"`
+	URL                      string     `url:"url,omitempty" json:"url,omitempty"`
+	GroupID                  int        `url:"group_id,omitempty"  json:"group_id,omitempty"`
+	PushEvents               bool       `url:"push_events,omitempty"  json:"push_events,omitempty"`
+	IssuesEvents             bool       `url:"issues_events,omitempty"  json:"issues_events,omitempty"`
+	ConfidentialIssuesEvents bool       `url:"confidential_issues_events,omitempty"  json:"confidential_issues_events,omitempty"`
+	MergeRequestsEvents      bool       `url:"merge_requests_events,omitempty"  json:"merge_requests_events,omitempty"`
+	TagPushEvents            bool       `url:"tag_push_events,omitempty"  json:"tag_push_events,omitempty"`
+	NoteEvents               bool       `url:"note_events,omitempty"  json:"note_events,omitempty"`
+	JobEvents                bool       `url:"job_events,omitempty"  json:"job_events,omitempty"`
+	PipelineEvents           bool       `url:"pipeline_events,omitempty"  json:"pipeline_events,omitempty"`
+	WikiPageEvents           bool       `url:"wiki_page_events,omitempty"  json:"wiki_page_events,omitempty"`
+	EnableSslVerification    bool       `url:"enable_ssl_verification,omitempty"  json:"enable_ssl_verification,omitempty"`
+	CreatedAt                *time.Time `url:"created_at,omitempty" json:"created_at,omitempty"`
+}
```

Comment:
As this is a response struct, it doesn't need the `url` tags and the `omitempty` options here.